lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,40 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
```

Comment:
Yeah, its current place is not immediately obvious. The doc itself is a cross between a guide and a specification, so didn't fit neatly into "style-guide". Will muse on how to address its visibility.